### PR TITLE
GTEST: Adjust the number of iterations for some tests

### PR DESCRIPTION
**Pull request by dmitrygladkov**
## What

Adjusted the number of iterations for some tests using `ucs::test_time_multiplier`

## Why ?

Those tests are excessively complicated to run under valgrind and takes too long.

## How ?

1. Use `ucs::test_time_multipler` where it is needed
2. Add possibility to increase the multiplier

### CODE REVIEW

File: test/gtest/common/test_helpers.cc [hunk 1]
```diff
@@ -208,9 +208,9 @@ void watchdog_stop()
     pthread_mutex_destroy(&watchdog.mutex);
 }
 
-int test_time_multiplier()
+int test_time_multiplier(int multiplier)
```
Comment (brminich) [inline test/gtest/common/test_helpers.cc]: I'd not change this function.

File: test/gtest/ucp/test_ucp_wireup.cc [hunk 2]
```diff
@@ -471,7 +471,7 @@ UCS_TEST_P(test_ucp_wireup_1sided, stress_connect) {
     for (int i = 0; i < 30; ++i) {
         sender().connect(&receiver(), get_ep_params());
         send_recv(sender().ep(), receiver().worker(), receiver().ep(), 1,
-                  10000 / ucs::test_time_multiplier());
+                  10000 / ucs::test_time_multiplier(10));
```
Comment (brminich) [inline test/gtest/ucp/test_ucp_wireup.cc]: you can just change `10000` -> `1000`

File: test/gtest/common/test_helpers.cc (hunk 1 above)
Comment (dmitrygladkov) [reply test/gtest/common/test_helpers.cc]: removed

File: test/gtest/ucp/test_ucp_wireup.cc (hunk 2 above)
Comment (dmitrygladkov) [reply test/gtest/ucp/test_ucp_wireup.cc]: done

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9936/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9937/ for details (Mellanox internal link).

File: test/gtest/ucp/test_ucp_wireup.cc [hunk 3]
```diff
@@ -471,7 +471,7 @@ UCS_TEST_P(test_ucp_wireup_1sided, stress_connect) {
     for (int i = 0; i < 30; ++i) {
         sender().connect(&receiver(), get_ep_params());
         send_recv(sender().ep(), receiver().worker(), receiver().ep(), 1,
-                  10000 / ucs::test_time_multiplier());
+                  1000 / ucs::test_time_multiplier());
```
Comment (yosefe) [inline test/gtest/ucp/test_ucp_wireup.cc]: but we are also reducing the test time without valgrind?
how about `10000 / (ucs::test_time_multiplier()*ucs::test_time_multiplier())`

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9958/ for details (Mellanox internal link).

**Comment by dmitrygladkov** [comment]
@yosefe ok to merge?

**Pull request merged**